### DISCUSSION TITLE
Mapbox integration

### DISCUSSION THREAD

**Discussion by be-bird-lade** [webstudio-is/webstudio-community#186] [Q&A]
Is there a clean way to implement Mapbox in a page at the moment? I keep getting the "Minified React error #418" as soon as I include the mapbox-gl.js. I would like to display GeoJSON data from my Directus database on my page in order to be able to show routes. I thought Mapbox would be great for this, but at the moment it seems to have some problems with Webstudio, or am I missing something?

**Comment by kof** [comment]
try using "client only" setting on html embed, which will not render this server-side, this is needed for any client-side widget ....

**Comment by be-bird-lade** [comment]
Oh, right, I had completely forgotten about that setting. Thank you very much!
